addon/object/computed.js

```diff
@@ -1,10 +1,11 @@
 import Ember from 'ember';
-
 import {
   decoratedPropertyWithRequiredParams,
   decoratedPropertyWithOptionalCallback
 } from '../utils/decorator-macros';
 
+const { assert } = Ember;
```

Comment:
You can use the lovely [ember-modules-codemod](https://github.com/ember-cli/ember-modules-codemod) to make the change quickly 😸